app/styles/main.css

```diff
@@ -19,6 +19,11 @@
   background-image: none;
 }
 
+.faq b {
```

Comment:
The reason I use <b> is:

1. We do it at my place of work, and I was taught by an HTML wizard :dizzy: 
1. It means that you can use <b> throughout the app if you ever add more components that need it and not have to keep specifying it
1. <b> is used to denote something that has special styling and I thought this would be nice to start as a convention for hood.ie.

Totally happy if yourself and others feel this is the wrong approach, though.